src/scripts/main.js

```diff
@@ -3,6 +3,20 @@
 const people = require('./lib/people');
 
 // eslint-disable-next-line no-console
-console.log(people); // you can remove it
+// console.log(people); // you can remove it
 
-// write your code here
+const str = document.querySelector('.dashboard');
+
+people.forEach((person) => {
+  const age = person.died - person.born;
+  const century = Math.ceil(person.died / 100);
+
+  str.insertAdjacentHTML('beforeend', `
+    <td>${person.name}</td>
+    <td>${person.sex}</td>
```

Comment:
```suggestion
    <td>${person.sex}</td>
```
Давай тут будемо відображати не 'm` або 'f', а 'Male' або 'Female', як на прикладі в умові